src/cluster/placement/config.go

```diff
@@ -23,11 +23,11 @@ package placement
 import (
 	"time"
 
-	"gopkg.in/yaml.v2"
-
 	"github.com/m3db/m3/src/cluster/generated/proto/placementpb"
 	"github.com/m3db/m3/src/cluster/kv"
 	"github.com/m3db/m3/src/x/instrument"
+
+	"gopkg.in/yaml.v2"
```

Comment:
Is this ordering achievable automatically with goimports or similar? (The previous may not have been...).

Reply:
`gci write -s 'Std,Prefix(github.com/m3db/m3),Def'` - https://github.com/daixiang0/gci
annoyingly golangci-lint doesn't autofix this.
Goland does this as well if you configure it.